src/components/User/UserBadge.tsx

```diff
@@ -60,7 +62,11 @@ export function UserBadge({
 
   return user ? (
     <Wrapper
-      className={cs(style.container, style[`side-${avatarSide}`], className)}
+      className={cs(
+	{[style.container]: !hasLink, [style.default_font_styles]: !hasLink},
```

Comment:
@ciphrd to make overwrites work consistently I couldn't find a smarter solution then extracting them in their own class and add them on the container or the anchor conditionally. 

I believe that this solution has the least dom polution and works consistently through the places we use the `<UserBadge />`